<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGraphs</class>
 <widget class="QDialog" name="TestGraphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Test Graphs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="plotLuckBtn">
         <property name="toolTip">
          <string>This will show you how your luck distributes through a number of random rolls.</string>
         </property>
         <property name="text">
          <string>Plot Luck Rolls</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="plotDamageBtn">
         <property name="toolTip">
          <string>This will show you how your ROLL influences the damage output.</string>
         </property>
         <property name="text">
          <string>Plot Damage Rolls</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
